### DISCUSSION TITLE
Sonarr radarr sabnzbd webserve

### DISCUSSION THREAD

**Discussion by flnEGknEF** [MichaIng/DietPi#6857] [Q&A]
My sonarr, radarr and sabnzbd won't consistently load on my phone. From the client side, I go to the IP address: 192.xxx.xx.xx:PORT and the webserve won't consistently load. Yet, then I open my browser on the pi and go to the same IP address and port it works fine. Then going back to my phone, it's like the service "wakes up" and is accessible again for an unknown period of time. 
Sleep and all settings like that wee disabled from the pi. 
On the pi I'm not using localhost, I am going to the same IP and port as I use on my phone. 
Since this is across multiple services I wonder if anyone has any ideas? What logs to look at? What should I look for? Be explicit as possible if you have ideas. I'm just a little better than a noob but not by much.

**Comment by MichaIng** [comment]
> On the pi I'm not using localhost, I am going to the same IP and port as I use on my phone.

This does not make a difference. The hosts LAN IP basically always connects via loopback device, just like localhost/127.0.0.1.

So you mean you can access the services in case you accessed them from localhost recently, but after a while the Pi is not accessible anymore? Is it these services only, all three of them together, or do also pings not reach the Pi anymore? And what about SSH?

Probably there is some kind of power saving for the network adapter. Which Pi is it exactly? WiFi or Ethernet? Onboard or USB adapter?

**Reply by flnEGknEF** [reply]
Interestingly I remember dietpi dash always working, without fail from the phone/client device.

The services always run from the Pi, which could be because using the Pi wakes up whatever was sleeping because its being used. No good way to test that since touching it would essentially wake it to begin with, if thats the issue.

I dont use SSH

PiHole, Sonarr, Radarr and Sabnzbd and docker/portainer with one active container (the only services im running) become inaccessible on the phone client (interestingly dietpi dash always runs).

Pi 4
Wifi, ethernet disabled
ipv4 only, internal LAN only. for what its worth I also do not have access set up for outside my network away from home
Onboard wifi adapter
LAN IP address is static

Power management is set to never sleep and do nothing when any power button or anything is pressed.

**Reply by MichaIng** [reply]
Hmm, strange that the dashboard behaves differently. The services themselves have no sleeping mode, hence my idea that it is the network adapter.

What does this show?
```sh
iw dev wlan0 set power_save off
```
And is this command present in the `/etc/network/interfaces` file? `dietpi-config` checks whether above command works, and in case adds it to the config to have this power savings mode for the WiFi adapter disabled once the interface is brought up. Otherwise many WiFi adapters go into a sleeping mode after a while. This however should affect every network service, including the dashboard.

And when the services are not reachable, can you try to run:
```sh
ping IPaddress
```
from a command prompt of the client system? Ah, difficult from a phone, but probably you have another desktop system to test this.

I remember there was a more recent issue with EEE (Energy-Efficient Ethernet) on the RPi. But it seems to have been solved, and not relevant in your case with WiFi.

**Reply by flnEGknEF** [reply]
So in the setting to turn the display blank, I had the timer set to "never" go blank, but the toggle was still in the on position. I would think "never" would be good enough. Turns out after I toggled the switch to "off" I haven't noticed any sleep issues since.

**Reply by MichaIng** [reply]
Where do you see this setting? Also strange that display blanking affects the WiFi connectivity 🤔.
